src/api/namespaces/neftymarket/routes/filters.ts

```diff
@@ -10,7 +10,7 @@ import {getAttributeFiltersAction} from '../handlers/filters';
 
 export function filtersEndpoints(core: NeftyMarketNamespace, server: HTTPServer, router: express.Router): any {
     const { caching, returnAsJSON } = server.web;
-    router.get('/v1/schemas/:collection_name/attribute_value_filters', caching(), returnAsJSON(getAttributeFiltersAction, core));
+    router.all('/v1/schemas/:collection_name/attribute_value_filters', caching(), returnAsJSON(getAttributeFiltersAction, core));
```

Comment:
what's the difference between `get` and `all`?

Reply:
get allows only GET requests. All allows all methods.

Reply:
and why do we want all if this is filters?

Reply:
We had the same support in the previous version. It might not be needed but it breaks the current app if we don’t allow post.